common/static/common/js/spec/components/paging_spec.js

```diff
@@ -73,19 +73,19 @@ define([ "jquery", "common/js/spec_helpers/ajax_helpers", "URI",
                 describe("setPage", function () {
                     it('can set the current page', function () {
                         var requests = AjaxHelpers.requests(this);
-                        pagingView.setPage(0);
+                        pagingView.setPage(1);
```

Comment:
So `collection.currentPage` is the last page number requested from the API? And in the case of this test, it is zero indexed?